src/app/styles/empty-state.css

```diff
@@ -16,8 +15,8 @@
 }
 
 .empty-state__image {
-  height: 64px;
-  width: 64px;
+  height: 56px;
+  width: 56px;
```

Comment:
Any specific reasoning behind changing this? I think we should also fix the spacing here between the header and the success state card.
<img width="321" alt="Screen Shot 2020-02-09 at 2 28 51 PM" src="https://user-images.githubusercontent.com/7717963/74111651-2eeaed80-4b4b-11ea-9ae7-0bc8d8a303bf.png">

Reply:
Moved back to 64px.